packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -679,29 +679,33 @@ export default class AwsS3Multipart extends BasePlugin {
       protocol: 's3-multipart',
       size: file.data.size,
       metadata: file.meta,
-    })
+    }, options)
     return res.token
   }
 
   // NOTE! Keep this duplicated code in sync with other plugins
   // TODO we should probably abstract this into a common function
```

Comment:
Should this change also be added in the other plugins?

Reply:
Good catch, looks like it should indeed

https://github.com/transloadit/uppy/blob/643fd8d055b592dd45350080d09bd120459cc039/packages/@uppy/aws-s3/src/MiniXHRUpload.js#L96-L98
https://github.com/transloadit/uppy/blob/90f7fb91974502c29ce935a0ed4befdecd6b0f64/packages/@uppy/tus/src/index.js#L734-L736
https://github.com/transloadit/uppy/blob/2b855ce1a7b7a68cd39aab354ac0fe97ba4d4bbf/packages/@uppy/xhr-upload/src/index.js#L597-L599

Reply:
We keep falling for this every time. I'd think we should tackle this shared code sooner rather than later, ideally.

Reply:
Yes, we've discussed with Antoine this will be a priority.